// This file is part of KWIVER, and is distributed under the
// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.

syntax = "proto2";

package kwiver.protobuf;

message geo_point {
  optional double x = 1;
  optional double y = 2;
  optional double z = 3;

  // If the "crs" value is missing, then the point is considered empty.
  optional int32 crs = 4;
}
